uint32 x_obj	    # x coordinates of detected object
uint32 y_obj	    # y coordinates of detected object
uint32 r_obj	    # 		 radius of detected object